crates/octez/src/async/baker.rs

```diff
@@ -69,18 +66,6 @@ impl OctezBakerConfigBuilder {
         self
     }
 
-    pub fn with_node_and_client(
```

Comment:
what's the intention behind removing this? i think the motivation was to have it for convenicince. What if we pass 
```
with_node_and_client(node: OctezNode, client: OctezClient) 
```

Reply:
See https://github.com/jstz-dev/jstz/pull/612#discussion_r1816703016